rust/chains/hyperlane-cosmos/src/providers/grpc/tests.rs

```diff
@@ -0,0 +1,77 @@
+use std::str::FromStr;
+
+use url::Url;
+
+use hyperlane_core::config::OperationBatchConfig;
+use hyperlane_core::{ContractLocator, HyperlaneDomain, KnownHyperlaneDomain};
+
+use crate::address::CosmosAddress;
+use crate::grpc::{WasmGrpcProvider, WasmProvider};
+use crate::{ConnectionConf, CosmosAmount, RawCosmosAmount};
+
+#[ignore]
```

Comment:
Nice, feels like a happy medium between not having unit tests include fallible network interactions & still being able to test things if issues arise

Wonder if there's an argument for something closer to integration tests in the future where we'd spin up a local cosmos node or something

Reply:
I would support adding integration tests with isolated environment.